python/paddle/static/io.py

```diff
@@ -98,8 +107,10 @@ def _clone_var_in_block(block, var):
 
 
 def prepend_feed_ops(
-    inference_program, feed_target_names, feed_holder_name='feed'
-):
+    inference_program: Program,
+    feed_target_names: list[str],
```

Comment:
```suggestion
    feed_target_names: Sequence[str],
```

输入的地方如果没有特殊要求（如 list 的 `append` 方法，或者用 `isinstance` 进行判断等），最好用 `Sequence` 吧 ～

后面有几个也一样 ～